packages/react-charts/src/components/ChartCursorTooltip/ChartCursorTooltip.tsx

```diff
@@ -159,7 +158,7 @@ export interface ChartCursorTooltipProps extends ChartTooltipProps {
    * values. If this prop is not provided it will be determined from the sign of the datum, and the value of the
    * horizontal prop.
    */
-  orientation?: OrientationTypes | VictoryNumberCallback;
+  orientation?: OrientationOrCallback;
```

Comment:
https://github.com/FormidableLabs/victory/commit/439e0ad8de23d1d6ed5580a90f60feaca3f09c4e#diff-1d0ded8dfdfbe48d50e6a99272c7548f81143f224f88b6528023cc87a32e9a40R115
Here they give a new interface named `OrientationOrCallback`, which can be used directly here.

Reply:
`OrientationOrCallback` should be equivalent to the old types

Reply:
Oh actually they are different because of the callback function. It will not return a `number` in the callback function, but returns `OrientationTypes` instead.